src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +11,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        if (userService.findByEmail(email) == null) {
+            return false;
+        }
+        if (userService.findByEmail(email).getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
Save user for variable that you get from `userService.findByEmail(email)` and after this check for password